tests/test_manifest.py

```diff
@@ -0,0 +1,46 @@
+"""Test ltdmason.manifest."""
+
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+from builtins import *  # NOQA
+from future.standard_library import install_aliases
+install_aliases()
+
+import pkg_resources
+import pytest
+
+from ltdmason.manifest import Manifest
+
+
+@pytest.fixture
+def demo_science_pipelines_manifest():
+    resource_args = (__name__, 'demo_science_pipelines_manifest.yaml')
+    assert pkg_resources.resource_exists(*resource_args)
+    yaml_data = pkg_resources.resource_string(*resource_args)
+    return yaml_data
+
+
+@pytest.mark.xfail
+def test_manifest_roundtrip(demo_science_pipelines_manifest):
+    # Can't get perfect round-trip equality of the YAML
```

Comment:
The assertion is passing.  Does `ruamel` have special semantics for equality testing?